### PR TITLE
Fix the bug that TestCase adhoc test result is not reported as failure

### PR DESCRIPTION
**Pull request by srz-zumix**
Fix the bug that TestCase adhoc test result is not reported as failure

### CODE REVIEW

**Comment by gennadiycivil** [comment]
CI builds failed .

**Comment by srz-zumix** [comment]
Where should I write a test that should fail?

**Comment by gennadiycivil** [comment]
Please check the status of travis and/or appveyor build, find the iszues amd address them . Thank you

**Comment by gennadiycivil** [comment]
209471987

File: googletest/test/gtest_setuptestcase_failure_test.cc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// Copyright 2009 Google Inc.  All rights reserved.
```
Comment (gennadiycivil) [inline googletest/test/gtest_setuptestcase_failure_test.cc]: What is the reason to remove this test from gtest_unittest and make is standalone? There are multiple changes needed to support it. Not clear what benefit it provides? Thank you

Comment (srz-zumix) [reply googletest/test/gtest_setuptestcase_failure_test.cc]: 1. FAIL on AdHocTestResultTest::SetUpTestCase
1. expect RUN_ALL_TESTS return 1
1. gtest_unittest expect RUN_ALL_TESTS return 0

If there is another test that expects RUN_ALL_TESTS to return 1, it is appropriate to write it there.

**Comment by gennadiycivil** [comment]
@srz-zumix Looking at the change again there appears to be a logical disconnect that caused you to have to move the AdHocTestResultTest fixture to the separate file. To test, simply break one of the tests ( for example change the EXPECT_TRUE to EXPECT_FALSE in AdHocTestResultForTestCaseShowsFailure. 
You will see something like

[  FAILED  ] AdHocTestResultTest.AdHocTestResultForTestCaseShowsFailure (9 ms)
[ RUN      ] AdHocTestResultTest.AdHocTestResultTestForUnitTestDoesNotShowFailure
[       OK ] AdHocTestResultTest.AdHocTestResultTestForUnitTestDoesNotShowFailure (2 ms)
[----------] 2 tests from AdHocTestResultTest (11 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 1 test case ran. (26 ms total)
[  PASSED  ] 1 test.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] AdHocTestResultTest.AdHocTestResultForTestCaseShowsFailure

 1 FAILED TEST
================================================================================
....
INFO: Build completed successfully, 5 total actions
INFO: Still uploading to http://sponge/2bd1923a-7e2d-457b-a7a0-b7211b806507
iblaze 14:25:36 blaze test succeeded.

Note that the build reports to have completed successfully even though one of the tests fail. 

Please take another look, the idea to correctly report failure in SetUpTestCase is worth pursuing.

**Comment by srz-zumix** [comment]
thank you for reviewing!

**Comment by gennadiycivil** [comment]
Looks stale, closing hank you

**Pull request closed without merging**